conf/application.conf

```diff
@@ -41,10 +41,9 @@ metrics {
   durationUnit = SECONDS
   showSamples = true
   jvm = true
-  enabled = true # disabled for local dev, should be enabled otherwise
+  enabled = false
```

Comment:
make enabled = true and add the comment line # disabled for local dev, should be enabled otherwise